src/input_xml.F90

```diff
@@ -125,6 +127,43 @@ subroutine read_settings_xml()
            path_output = trim(path_output) // "/"
     end if
 
+    ! Check for a trigger node and get trigger information
+    if (check_for_node(doc, "trigger")) then
+      call get_node_ptr(doc, "trigger", node_trigger)
+
+      ! Get trigger status
+      call get_node_value(node_trigger, "status", temp_str)
```

Comment:
I still find "status" very unintuitive. Can we change this to "active" or "on" or something that more clearly indicates one is turning the trigger on?